.github/workflows/snapshot.yml

```diff
@@ -0,0 +1,78 @@
+# ------------------------------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
```

Comment:
At this point we have added too many yaml files here which are very similar, its a good time to merge these.

Reply:
Second that.

Reply:
How would you merge these?  I don't think these can be templatized w/o helm.

Reply:
@LMWF we can use the single workflow. but you need to add the step to parse changed app. I can sync up with you.

Reply:
Opened

Reply:
Sure, lets use the new issue to track these improvements, in the interest of getting the first run up and running, I am fine checking it in and track the improvement in the new issue, but lets try to tackle them in this milestone.